lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -310,4 +310,10 @@ def self.ae_user_object(options = {})
       $miq_ae_logger.info("User [#{obj.userid}] with current group ID [#{obj.current_group.id}] name [#{obj.current_group.description}]")
     end
   end
+
+  V2_PASSWORD_REGEXP = /v([0-9]+)%3A%7B(.*?)%7D/
+
+  def self.sanitize_string(str)
```

Comment:
I would recommend changing this name to be more specific since it is really handling the special use-case of an encoded string.

Maybe: `sanitize_encoded_string`

@bdunne Do you think this should live in MiqPassword?  A bit of a special case.